CONTRIBUTING.md

```diff
@@ -1,42 +1,67 @@
+* Feel free to post your question on our [community forum](community-forum) or join our [matrix community chat][matrix-support].
 
-2. Ensure any install or build dependencies are removed before the end of the layer when doing a
-   build.
+### Did you find a bug?
 
-3. Update the README.md with details of changes to the interface, this includes new environment
-   variables, exposed ports, useful file locations and container parameters.
+* **Ensure the bug wasn't already reported** by searching on GitHub under [Issues](issues).
 
-4. Increase the version numbers in any examples files and the README.md to the new version that this
-   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
+* If you're unable to find an open issue addressing the problem, [open a new one](new_issue). Be sure to use one of the templates we provide if your request applies to them.
 
-5. You may merge the Pull Request in once you have the sign-off of two other developers, or if you
-   do not have permission to do that, you may request the second reviewer to merge it for you.
+### Did you write a patch that fixes a bug?
 
-## Sign your work
+* Open a new GitHub pull request with the patch. See the section [submitting a pull request](#submitting-a-pull-request) for details on this.
+
+* Ensure the PR description is precise about the problem and your solution. Just fill out our template. That should cover the most important information.
+
+### Do you intend to add a new feature or change an existing one?
+
+* Suggest your idea via a new GitHub issue. After a confirmation about your idea, you can start writing code. Our maintainers and other project developers can provide useful details about the architecture and show you relevant issues and discussions.
+
+### Do you want to work on translations?
+
+* If you want to improve a translation or add a new translation altogether, we handle those via [POEditor](poeditor).
+
+## Certificate of Origin
+
+By contributing to this project you agree to the [Developer Certificate of
+Origin (DCO)](docs/legal/developer-certificate-of-origin.txt). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution. 
+The DCO is a legally binding statement, please [read it carefully](docs/legal/developer-certificate-of-origin.txt).
 
-We use the Developer Certificate of Origin (DCO) as a additional safeguard
-for the HedgeDoc project. This is a well established and widely used
-mechanism to assure contributors have confirmed their right to license
-their contribution under the project's license.
-Please read [docs/legal/developer-certificate-of-origin.txt][dcofile].
 If you can certify it, then just add a line to every git commit message:
 
-```git
+```
   Signed-off-by: Random J Developer <random@developer.example.org>
 ```
 
 Use your real name (sorry, no pseudonyms or anonymous contributions).
```

Comment:
"Please use your real name, mr. random developer". Can we replace the example name with a more serious one?

Reply:
done